### ISSUE TITLE
CLI tool for RPM metadata generators

### ISSUE THREAD

**Issue by g4s8** [artipie/rpm-adapter#68]
Let's create simple CLI tool to generate metadata for `rpm` packages. We need some class for this CLI with `main()` method (don't forget to define main method in `pom.xml`). It will be a wrapper for `Rpm` class. The syntax of this tool can be:
```
java -jar rpm-adapter.jar --naming-policy=simple --digest=SHA256 update ./package.rpm
# or
java -jar rpm-adapter.jar batchUpdate ./repo-dir/
```
So there are some options, one command, and one (or more) required positional arguments. 

Options are:
 - `naming-policy` - (optional, default `simple`) configures `NamingPolicy` for `Rpm`
 - `digest` - (optional, default `sha256`) configures `Digest` instance for `Rpm`

Commands are:
 - `update` - update one or more `.rpm` packages. It's equivalent of `Rpm#update` call
 - `batchUpdate` - update specified directory as RPM repository, same as `Rpm#batchUpdate` call

**Comment by ammaratef45** [comment]
@0crat wait for the review for PR #72

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#89](https://github.com/artipie/rpm-adapter/issues/89) is still not solved.

**Comment by g4s8** [comment]
@ammaratef45 done in #72

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
> @0crat quality good  ([here](https://github.com/artipie/rpm-adapter/issues/68#issuecomment-611701376))

@sereshqua There is no quality review for [#68](https://github.com/artipie/rpm-adapter/issues/68), no performer

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#113](https://github.com/artipie/rpm-adapter/issues/113), [#114](https://github.com/artipie/rpm-adapter/issues/114) are still not solved; solved: [#89](https://github.com/artipie/rpm-adapter/issues/89).

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#113](https://github.com/artipie/rpm-adapter/issues/113), [#176](https://github.com/artipie/rpm-adapter/issues/176) are still not solved; solved: [#114](https://github.com/artipie/rpm-adapter/issues/114), [#89](https://github.com/artipie/rpm-adapter/issues/89).

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#176](https://github.com/artipie/rpm-adapter/issues/176), [#205](https://github.com/artipie/rpm-adapter/issues/205) are still not solved; solved: [#113](https://github.com/artipie/rpm-adapter/issues/113), [#114](https://github.com/artipie/rpm-adapter/issues/114), [#89](https://github.com/artipie/rpm-adapter/issues/89).

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#176](https://github.com/artipie/rpm-adapter/issues/176) is still not solved; solved: [#113](https://github.com/artipie/rpm-adapter/issues/113), [#114](https://github.com/artipie/rpm-adapter/issues/114), [#205](https://github.com/artipie/rpm-adapter/issues/205), [#89](https://github.com/artipie/rpm-adapter/issues/89).

**Comment by 0pdd** [comment]
@g4s8 all 5 puzzles are solved here: [#113](https://github.com/artipie/rpm-adapter/issues/113), [#114](https://github.com/artipie/rpm-adapter/issues/114), [#176](https://github.com/artipie/rpm-adapter/issues/176), [#205](https://github.com/artipie/rpm-adapter/issues/205), [#89](https://github.com/artipie/rpm-adapter/issues/89).
